kurtosis-devnet/pkg/devnet/images/repository.go

```diff
@@ -0,0 +1,37 @@
+package images
+
+import "fmt"
+
+// Repository maps component versions to their corresponding Docker image URLs
+type Repository struct {
+	mapping map[string]string
+}
+
+// NewRepository creates a new Repository instance with predefined mappings
+func NewRepository() *Repository {
+	return &Repository{
+		mapping: map[string]string{
+			// op-stack components
+			"op-deployer":   "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-deployer:%s",
+			"op-geth":       "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:%s",
+			"op-node":       "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-node:%s",
+			"op-batcher":    "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-batcher:%s",
+			"op-proposer":   "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-proposer:%s",
+			"op-challenger": "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-challenger:%s",
+
+			// alternative clients
```

Comment:
We should not enshrine the concept of alternative or else one codebase will be at an advantage vs another

Reply:
ack, fixed !